spec/forms/flood_risk_engine/steps/local_authority_address_form_spec.rb

```diff
@@ -58,35 +40,33 @@ def form_requires_address_lookup
         }
 
         it "is valid when valid UK UPRN supplied via drop down but no street_address present" do
-          form_requires_address_no_street_lookup do
-            expect(form.validate(valid_but_no_street_address)).to eq true
-          end
+          mock_ea_address_lookup_find_by_uprn
+          expect(form.validate(valid_but_no_street_address)).to eq true
```

Comment:
I need to dig deeper to be 100% but I'm concerned that here we are no longer exercising the correct underlying test. AFAIK the data returned by the mock is always the same postcode/data set (which will validate) but this test was to check the situation where post code valid e.g BS9 9XX  but  the subsequently returned address results set is empty